src/rsz/README.md

```diff
@@ -7,66 +7,66 @@ utilization is exceeded.
 
 ## Commands
 
-### Set Wire RC
-
-```
-set_wire_rc [-clock] [-signal]
-            [-layer layer_name]
-            [-resistance res]
-            [-capacitance cap]
+```{note}
+- Parameters in square brackets `[-param param]` are optional.
+- Parameters without square brackets `-param2 param2` are required.
 ```
 
+### Set Wire RC
+
 The `set_wire_rc` command sets the resistance and capacitance used to estimate
 delay of routing wires.  Separate values can be specified for clock and data
 nets with the `-signal` and `-clock` flags. Without either `-signal` or
 `-clock` the resistance and capacitance for clocks and data nets are set.
 Use `-layer` or `-resistance` and `-capacitance`.  If `-layer` is used,
 the LEF technology resistance and area/edge capacitance values for the
-layer are used for a default width wire on the layer.  The resistance and
-capacitance values are per length of wire, not per square or per square micron.
-The units for `-resistance` and `-capacitance` are from the first Liberty
-file read, resistance_unit/distance_unit (typically kohms/micron) and
-Liberty capacitance_unit/distance_unit (typically pf/micron or ff/micron).
-If distance units are not specified in the Liberty file, microns are used.
+layer are used for a default width wire on the layer.  
+
+```tcl
+set_wire_rc 
+    [-clock] 
+    [-signal]
+    [-layer layer_name]
+    [-resistance res]
+    [-capacitance cap]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-clock` | enable setting of RC for clock nets |
+| `-signal` | enable setting of RC for signal nets | 
+| `-layer` | if used, the LEF default RC values are used. |
+| `-resistance` | resistance per unit length, units are from the first Liberty file read, usually in the form of $\frac{resistanceUnit}{distanceUnit}$. Usually kΩ/µm. |
 
 ### Set Layer RC
 
 The `set_layer_rc` command can be used to set the resistance and capacitance
 for a layer or via. This is useful if these values are missing from the LEF file,
 or to override the values in the LEF.
 
+```tcl
+set_layer_rc 
+    [-layer layer]
+    [-via via_layer]
+    [-capacitance cap]
+    [-resistance res]
```

Comment:
Might as well order it resistance then capacitance to match order of `set_wire_rc`